From: "Matthew Wilcox (Oracle)" <willy@infradead.org>
Subject: afs: do not test the return value of folio_start_writeback()
Date: Wed, 8 Nov 2023 20:46:03 +0000

In preparation for removing the return value entirely, stop testing it
in afs.

Link: https://lkml.kernel.org/r/20231108204605.745109-3-willy@infradead.org
Signed-off-by: Matthew Wilcox (Oracle) <willy@infradead.org>
Cc: David Howells <dhowells@redhat.com>
Cc: Steve French <sfrench@samba.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 fs/afs/write.c |    6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

--- a/fs/afs/write.c~afs-do-not-test-the-return-value-of-folio_start_writeback
+++ a/fs/afs/write.c
@@ -559,8 +559,7 @@ static void afs_extend_writeback(struct
 
 			if (!folio_clear_dirty_for_io(folio))
 				BUG();
-			if (folio_start_writeback(folio))
-				BUG();
+			folio_start_writeback(folio);
 			afs_folio_start_fscache(caching, folio);
 
 			*_count -= folio_nr_pages(folio);
@@ -595,8 +594,7 @@ static ssize_t afs_write_back_from_locke
 
 	_enter(",%lx,%llx-%llx", folio_index(folio), start, end);
 
-	if (folio_start_writeback(folio))
-		BUG();
+	folio_start_writeback(folio);
 	afs_folio_start_fscache(caching, folio);
 
 	count -= folio_nr_pages(folio);
_
